bintray_test.go

```diff
@@ -529,3 +529,17 @@ func deleteBintrayRepo() {
 		os.Exit(1)
 	}
 }
+
+func TestMavenSync(t *testing.T) {
```

Comment:
This test seems to doing nothing.
I assime the ```bintrayCli.Exec("maven-sync", versionPath)``` call returns an err, which is not checked.
I suggest removing this test. I think we can rely on the Maven Central Sync test in the *jfroig-client-go* library.